### PR TITLE
Create contest

### PR DESCRIPTION
**Pull request by banh0006**
- Added Navbar (just temporary).
- Added Create Contest Page.

Screen Shot 2021-01-12 at 9 27 04 PM
Screen Shot 2021-01-12 at 9 27 09 PM

### CODE REVIEW

File: client/src/components/CreateContestForm.js [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+                        id="amount"
+                        value={amount}
+                        error={amountError}
+                        helperText={amountHelperText}
+                        onChange={onAmountChange}
+                        className={classes.inputMargin}
+                        type="number"
+                        InputProps={{
+                            startAdornment: <InputAdornment position="start">$</InputAdornment>,
+                            // endAdornment: <InputAdornment position="end">.00</InputAdornment>,
```
Comment (bonnieli) [inline client/src/components/CreateContestForm.js]: delete if not used

File: client/src/components/CreateContestForm.js [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+                        }}
+                        variant="outlined"
+                    />
+                </Grid>
+                <Grid item xs={3}>
+                    <div className={classes.borderedDiv}>
+                        <TextField
+                            id="deadlineDate"
+                            defaultValue="2021-01-12"
+                            inputProps={{ style: { padding: "1rem" } }}
```
Comment (bonnieli) [inline client/src/components/CreateContestForm.js]: this input style seems to be used often thorughout this component, can you create a style and pass in a className?

**Review by bonnieli — APPROVED** [review]
this looks great use of functional components and classes! you can take this out of draft PR and if you're happy with it can merge it in

File: client/src/components/CreateContestForm.js (hunk 2 above)
Comment (banh0006) [reply client/src/components/CreateContestForm.js]: It's quite difficult for me to figure out how to make a style with "inputProps" attribute. Do you have any suggestions? Or maybe I'll do it later.

**Pull request merged**